hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/common/tagdetails/AbstractTargetTagToken.java

```diff
@@ -45,13 +45,13 @@ protected AbstractTargetTagToken(final SpPermissionChecker checker, final I18N i
     }
 
     @EventBusListenerMethod(scope = EventScope.UI)
-    void onEventTargetTagCreated(final TargetTagCreatedEventContainer container) {
+    public void onEventTargetTagCreated(final TargetTagCreatedEventContainer container) {
```

Comment:
Is public required? This method is only called by Spring, so I'd prefere to reduce visibility